plugins/apache-airflow/src/components/DagTableComponent/DagTableComponent.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Progress,
+  StatusError,
+  StatusOK,
+  Table,
+  TableColumn,
+} from '@backstage/core-components';
+import { useApi } from '@backstage/core-plugin-api';
+import Box from '@material-ui/core/Box';
+import Chip from '@material-ui/core/Chip';
+import IconButton from '@material-ui/core/IconButton';
+import Switch from '@material-ui/core/Switch';
+import Tooltip from '@material-ui/core/Tooltip';
+import Typography from '@material-ui/core/Typography';
+import OpenInBrowserIcon from '@material-ui/icons/OpenInBrowser';
+import Alert from '@material-ui/lab/Alert';
+import React from 'react';
+import { useAsync } from 'react-use';
+import { apacheAirflowApiRef } from '../../api';
+import { Dag } from '../../api/types';
+import { ScheduleIntervalLabel } from '../ScheduleIntervalLabel';
+
+type DagTableRow = Dag & {
+  id: string;
+  dagUrl: string;
+};
+
+const columns: TableColumn[] = [
+  {
+    title: 'Paused',
+    field: 'is_paused',
+    render: (row: Partial<DagTableRow>) => (
+      <Tooltip title="Pause/Unpause DAG">
+        <Switch checked={!row.is_paused} disabled />
+      </Tooltip>
+    ),
+    width: '5%',
+  },
+  {
+    title: 'DAG',
+    field: 'id',
+    render: (row: Partial<DagTableRow>) => (
+      <div>
+        <Typography variant="subtitle2" gutterBottom noWrap>
+          {row.id}
+        </Typography>
+        <Box display="flex" alignItems="center">
+          {row.tags?.map((tag, ix) => (
+            <Chip label={tag.name} key={ix} size="small" />
+          ))}
+        </Box>
+      </div>
+    ),
+    width: '60%',
+  },
+  {
+    title: 'Owner',
+    field: 'owners',
+    render: (row: Partial<DagTableRow>) => (
+      <Box display="flex" alignItems="center">
+        {row.owners?.map((owner, ix) => (
+          <Chip label={owner} key={ix} size="small" />
```

Comment:
Feature request (😬): Would it be possible to make the chip clickable and try to resolve it to a user/group entity in the catalog? I could see how integrators (us included) would have team names as DAG owners, so it would be nice to provide a jump-click functionality to the team's overview page

Reply:
I like this idea! I've gone ahead and added a [section in the README](https://github.com/cmpadden/backstage/blob/plugin-apache-airflow/plugins/apache-airflow/README.md) to keep track of feature requests, and I've added an entry for this. For this initial pull request I just wanted to get a MVP out there, but I definitely plan on adding more features in the future. Thanks for the suggestion.